### PR TITLE
util: add more unit test

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
Improve `util` pkg unit test coverage
```
▶ go test -cover
...
OK: 5 passed                                                                                                                                                                                               │▶ make fmt
PASS                                                                                                                                                                                                       │gofmt (simplify)
coverage: 90.8% of statements                                                                                                                                                                              │util/prefix_helper_test.go
ok      github.com/pingcap/tidb/util    0.025s


tidb/util/execdetails                                                                                                                                                        
▶ go test -coverprofile=coverage.out
PASS
coverage: 97.7% of statements
ok      github.com/pingcap/tidb/util/execdetails        0.004s

tidb/util/filesort                                                                                                                                          
▶ go test -coverprofile=coverage.out
...
OK: 6 passed
PASS
coverage: 86.4% of statements
ok      github.com/pingcap/tidb/util/filesort   0.939s

tidb/util/format                                                                                                                                                                  
▶ go test -coverprofile=coverage.out
OK: 1 passed
PASS
coverage: 83.6% of statements
ok      github.com/pingcap/tidb/util/format     0.004s

```

### CODE REVIEW

File: util/filesort/filesort_test.go [hunk 1]
```diff
@@ -181,7 +181,26 @@ func (s *testFileSortSuite) TestMultipleFiles(c *C) {
...
+
+	// Test for basic function.
+	_, err = fsBuilder.Build()
+	c.Assert(err.Error(), Equals, "StatementContext is nil")
+	fsBuilder.SetSC(sc)
+	_, err = fsBuilder.Build()
+	c.Assert(err.Error(), Equals, "key size is not positive")
+	fsBuilder.SetDesc(byDesc)
+	_, err = fsBuilder.Build()
+	//fsBuilder.SetSchema(keySize,valSize).SetDesc(byDesc)
```
Comment (zimulala) [inline util/filesort/filesort_test.go]: Could we this line？

Comment (crazycs520) [reply util/filesort/filesort_test.go]: done. Thanks.

File: util/misc_test.go [hunk 2]
```diff
@@ -113,3 +117,58 @@ func (s *testMiscSuite) TestCompatibleParseGCTime(c *C) {
...
+	c.Assert(row[2], Equals, pi.Host)
+	c.Assert(row[3], Equals, pi.DB)
+	c.Assert(row[4], Equals, "Sleep")
+	c.Assert(row[5], Equals, uint64(0))
+	c.Assert(row[6], Equals, "1")
+	c.Assert(row[7], Equals, "test")
+
+	// Test for RandomBuf.
+	buf := RandomBuf(5)
+	c.Assert(len(buf) == 5, IsTrue)
```
Comment (bb7133) [inline util/misc_test.go]:
```suggestion
	c.Assert(len(buf), Equals, 5)
```

File: util/filesort/filesort_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (crazycs520) [reply util/filesort/filesort_test.go]: done.

**Comment by bb7133** [comment]
LGTM

**Pull request merged**